<launch>

  <arg name="vehicle_name" default="X1"/>

  <node name="path_follower" type= "path_follower_node" pkg="marble_guidance" output="screen">

    <remap from="path_follower/odometry"        to="odometry_map"/>
    <remap from="path_follower/path"            to="ma_goal_path"/>
    <remap from="path_follower/cmd_vel"         to="cmd_vel"/>
    <remap from="path_follower/enable_backup"   to="enable_backup"/>

    <param name="vehicle_name"                    value="$(arg vehicle_name)"/>
    <param name="loop_rate"                       value="20.0"/>

    <param name="yawrate_k0"                      value="1.0"/>
    <param name="yawrate_kd"                      value="1.0"/>
    <param name="max_yawrate"                     value="0.5"/>
    <param name="max_forward_speed"               value="1.0"/>
    <param name="turn_in_place_thresh"            value="1.0"/>
    <param name="turn_in_place_yawrate"           value="0.25"/>
    <param name="lookahead_distance_threshold"    value="1.0"/>
    <param name="enable_debug"                    value="1"/>

  </node>

</launch>
